src/main/java/nextstep/subway/line/domain/Distance.java

```diff
@@ -18,23 +21,23 @@ public Distance(int distance) {
 
     private void validateDistance(int distance) {
         if (distance <= 0) {
-            throw new IllegalArgumentException("역의 거리는 양수를 입력해야합니다.");
+            throw new IllegalArgumentException(DISTANCE_CANNOT_BE_ZERO.getMessage());
         }
     }
 
 
-    public void minus(Distance distance) {
-        if (this.distance <= distance.getDistance()) {
-            throw new IllegalArgumentException("역과 역 사이의 거리보다 좁은 거리를 입력해주세요");
+    public void minus(int distance) {
```

Comment:
```suggestion
     public void minus(Distance distance) {
       ...
       this.distance -= distance.distance;
```

이런패턴으로 한다면, `minus` 호출하는 부분에서 좀 더 간단해질수 있겠네요 :thinking: